docs/src/pages/index.tsx

```diff
@@ -204,7 +203,7 @@ export const IndexPage = (): React.ReactElement<any> => (
                                 description="Check out our components & documentation."
                                 imgSrc="/images/developer-icon.svg"
                                 title="Developers"
-                                to="/components"
+                                to="/install"
```

Comment:
Defaulting to install page? Hmm

Reply:
Since the site is slowly growing into two sections (theming and components) I felt the Install page was a good one to land on. Install instructions, explanation about xstyled, suggestions about what to do next.